### DISCUSSION TITLE
A dynamic Overlay Position for Notification - or mouse pointer and prompt indicator integration

### DISCUSSION THREAD

**Discussion by KodeXall** [jonaskohl/CapsLockIndicator#130] [Ideas]
I an a user of a 38" monitor with an aspect ratio of 21:9. In the future I will also be using such a widescreen, but bigger. I also often write when laying in bed, thus I need to zoom in my view [with Windows 10 Magnifier]. I used to write then with an onscreen [i.e software] keyboard but now I do it with a mini handheld [i.e hardware] keyboard

All these factors combined make the current Notification feature not sufficient enough. To be sure that I always am informed of an e.g. CapsLock being turned on I would need to set the size of the font to a bloated one thus see a huge Notification that would take most of my screen [so that when zoomed in even 300% I would still see it]. Such workaround could work but it would not be pretty

Instead of that there could be other solutions added to help this issue / modus operandi:

**A]** Make the current Notification follow the pointer of the mouse

**B]** Make the current Notification follow various prompt signs in a focused window / box

**C]** Add another customizable indicator that would follow the pointer of the mouse

**D]** Add another customizable indicator that would follow various prompt signs in focused windows / boxes

**E]** Change the pointer of the mouse to a different one or add a customizable shadow / background for it

**F]** Change the various prompt signs to a different ones or add a customizable shadow / background for them

I reckon that option F is pretty much impossible to implement [because it would also need to take care / over of e.g. Power Shell command line]. But I think C, D and E would be doable, with C being an optimal on; as long as it would not be mandatory and allow for setting of the exact position in relation to object in question [so that it would accommodate needs of people using alphabets written not only from left to right but also right to left and top to bottom

**Comment by jonaskohl** [comment]
I have been experimenting with a dynamically positioned overlay for some time now, but was never quite satisfied with the implementations as they either offered limited flexibility or suffered from poor UX. But I'll keep working on this feature
